src/modules/LanItems.rb

```diff
@@ -485,6 +485,9 @@ def update_item_udev_rule!(based_on = :mac)
     # @param new_val     [string] value for new key
     # @return updated rule when replace_key is found, current rule otherwise
     def ReplaceItemUdev(replace_key, new_key, new_val)
+      Items()[@current]["udev"] = { "net" => [] } if !Items()[@current]["udev"]
```

Comment:
It is just a save check, and was part of the method just moved to the beginning and used and empty array instead of a hash as the default value. 

If I remember correctly there was some issue in the past because of not ["udev"] when renaming , removed or something like that so... 

as for example check added in https://github.com/yast/yast-network/pull/284 or bug https://bugzilla.suse.com/show_bug.cgi?id=914833 I could assume that, but I'm being conservative.